src/ray/gcs/gcs_client/global_state_accessor.cc

```diff
@@ -262,6 +262,16 @@ std::unique_ptr<std::string> GlobalStateAccessor::GetPlacementGroupByName(
   return placement_group_table_data;
 }
 
+std::unique_ptr<std::string> GlobalStateAccessor::GetInternalKV(const std::string &key) {
+  std::string value;
+  Status status = gcs_client_->InternalKV().Get(key, value);
+  if (status.ok()) {
+    return std::make_unique<std::string>(value);
+  } else {
+    return nullptr;
+  }
```

Comment:
How about simplifying to:
return status.ok() ? std::make_unique<std::string>(value) : nullptr;

Reply:
sure.